Gemfile.lock

```diff
@@ -12,42 +12,40 @@ GEM
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-    addressable (2.4.0)
+    addressable (2.7.0)
+      public_suffix (>= 2.0.2, < 5.0)
     arel (9.0.0)
     ast (2.4.2)
     backports (3.6.8)
     bcrypt (3.1.11)
-    byebug (2.7.0)
-      columnize (~> 0.3)
-      debugger-linecache (~> 1.2)
-    capybara (2.6.2)
+    byebug (11.1.3)
+    capybara (3.35.3)
       addressable
-      mime-types (>= 1.16)
-      nokogiri (>= 1.3.3)
-      rack (>= 1.0.0)
-      rack-test (>= 0.5.4)
-      xpath (~> 2.0)
-    coderay (1.1.1)
-    columnize (0.9.0)
-    concurrent-ruby (1.1.5)
-    coveralls (0.8.13)
-      json (~> 1.8)
-      simplecov (~> 0.11.0)
+      mini_mime (>= 0.1.3)
+      nokogiri (~> 1.8)
+      rack (>= 1.6.0)
+      rack-test (>= 0.6.3)
+      regexp_parser (>= 1.5, < 3.0)
+      xpath (~> 3.2)
+    coderay (1.1.3)
+    concurrent-ruby (1.1.8)
+    coveralls (0.8.23)
+      json (>= 1.8, < 3)
+      simplecov (~> 0.16.1)
       term-ansicolor (~> 1.3)
-      thor (~> 0.19.1)
-      tins (~> 1.6.0)
+      thor (>= 0.19.4, < 2.0)
+      tins (~> 1.6)
     crass (1.0.5)
-    debugger-linecache (1.2.0)
     delayed_job (4.1.8)
       activesupport (>= 3.0, < 6.1)
     delayed_job_active_record (4.1.0)
       activerecord (>= 3.0, < 5)
```

Comment:
@dperson have you by any chance been making manual edits to `Gemfile.lock` or selectively committing lines? There's some weird stuff happening when I try to add/change gems. Based on this line, it looks like you shouldn't have been able to update `activerecord` to `5.2`.

Reply:
The final version (from the big update you wanted split up) was updated by `bundle lock --update` but I was trying to do these updates by hand to update just the ones effected by this change. Did I fat finger something?

Reply:
It looks like #533 should not have been possible without also updating `delayed_job_active_record`. I'm a little surprised bundler let it happen. If you run `bundle update --group=test` on `master` it will throw an error. #546 should fix it.

It's not a huge deal, but in general, I would avoid manually updating `Gemfile.lock` or selectively adding lines via something like `git add -p`. Instead, it's best to let bundler resolve any updates to make sure everything is in a consistent state. I'm not familiar with `bundle lock --update`, but when I'm updating gems I'll general use some variation on [`bundle update`](https://bundler.io/man/bundle-update.1.html). You can use `bundle update group=test` or `bundle update --minor` or `bundle update activerecord`.

Reply:
Sorry, looks like you'll need to rebase this one.

Reply:
Ah, sorry about that. I'm not a ruby expert, and appreciate the pointer on `bundle`.

The rebase is complete. Thanks!

Reply:
Yeah, this kind of shakes my faith in Bundler. I poked around for a while but wasn't able to find a good way to get it to throw an error, aside from trying to add a new gem.